lib/tessel.js

```diff
@@ -5,9 +5,22 @@ var Emitter = require("events").EventEmitter;
 var tessel;
 var factory;
 
+/**
+ * dirName gets or sets the current value of `__dirname`
```

Comment:
Add an explanation for why we would ever want to set this, ie. testing and so forth